src/WebJobs.Script/Workers/Rpc/Configuration/RpcWorkerConfigFactory.cs

```diff
@@ -248,6 +248,7 @@ internal string GetHydratedWorkerPath(RpcWorkerDescription description)
             {
                 version = description.DefaultRuntimeVersion;
             }
+            _logger.LogDebug($"{RpcWorkerConstants.FunctionWorkerRuntimeVersionSettingName} for language {description.Language}: {version}");
```

Comment:
change to `_logger.LogDebug("EnvironmentVariable `{workerRuntimeVersionSettingName}: `{version}`", RpcWorkerConstants.FunctionWorkerRuntimeVersionSettingName, version);`

FunctionWorkerRuntimeVersionSettingName is not per runtime. It is an application level setting.